gtda/homology/simplicial.py

```diff
@@ -28,13 +28,12 @@ class VietorisRipsPersistence(BaseEstimator, TransformerMixin, PlotterMixin):
     :ref:`Vietoris–Rips filtrations
     <vietoris-rips_complex_and_vietoris-rips_persistence>`.
 
-    Given a :ref:`point cloud <finite_metric_spaces_and_point_clouds>` in
-    Euclidean space, or an abstract :ref:`metric space
-    <finite_metric_spaces_and_point_clouds>` encoded by a distance matrix,
-    information about the appearance and disappearance of topological features
-    (technically, :ref:`homology classes <homology_and_cohomology>`) of various
-    dimensions and at different scales is summarised in the corresponding
-    persistence diagram.
+    Given a :ref:`point cloud <TODO>` in Euclidean space, or an abstract
```

Comment:
Wait, I don't think it's true that we don't have point clouds or metric spaces...

Reply:
ouuups

Reply:
Actually, there was more to that - only persistence_image is missing